### DISCUSSION TITLE
Zoom plugin

### DISCUSSION THREAD

**Discussion by brieucdlf** [sgratzl/chartjs-chart-geo#136] [Q&A] [answered]
Hey guys!

Is it possible, to use the zoom plugin with chart-geo?
https://github.com/chartjs/chartjs-plugin-zoom

Thanks!

**Comment by sgratzl** [comment, marked as answer]
not it does not, this plugin doesn't use regular scales since the map has to be projected. However, the zoom plugin operates on scales.

**Comment by brieucdlf** [comment]
Thanks for your time and your anwser :)
